lib/adapters/http/index.js

```diff
@@ -950,7 +950,7 @@ function HttpPouch(opts, callback) {
             if (returnDocs) {
               results.results.push(c);
             }
-            utils.call(opts.onChange, c);
+            opts.onChange(c);
```

Comment:
Yeh the only point of `utils.call` is when we dont know whether the function is undefined, otherwise definitely isnt needed, a lot of places we use it then we ended up wrapping the function we call it from at a higher level and its no longer needed